### ISSUE TITLE
a11y: Author names should be available in the "latest updates" section of the homepage. [moved]

### ISSUE THREAD

**Issue by ebidel** [html5rocks/www.html5rocks.com#94]
This is [Issue 782](http://code.google.com/p/html5rocks/issues/detail?id=782) moved from the [old Google Code project](http://code.google.com/p/html5rocks/) and added by 2012-01-27T15:30:30.000Z by [mkwst@google.com](http://code.google.com/u/mkwst@google.com/). Please review that bug for more context and additional comments, but update this bug.

Original labels: Type-Bug, Priority-P2, a11y
### Original description

```
Hooray for not including the author's name in the JSON feed that's being loaded.
```
